docs/UserGuide.md

```diff
@@ -918,18 +918,29 @@ If this issue occurs, transfer the CSV file you want to import to the same folde
-| **Import**        | `import i\|FILEPATH` <br><br>• e.g. `import i\|C:/Users/alk/Downloads/avengersassemble.csv`                                                                                                                                                |
+| Action                 | Format, Examples                                                                                                                                                                                                                                        |
+|------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Help**               | `help`                                                                                                                                                                                                                                                  |
+| **Clear**              | `clear`                                                                                                                                                                                                                                                 |
+| **Import**             | `import i\|FILEPATH` <br><br>• e.g. `import i\|C:/Users/alk/Downloads/avengersassemble.csv`                                                                                                                                                             |
+| **Add**                | `add n\|NAME p\|PHONE_NUMBER e\|EMAIL a\|ADDRESS [t\|TAG]… [m\|MATRICULATION_NUMBER] [s\|STUDIO] [r\|REFLECTION]​` <br><br>• e.g. `add n\|James Ho p\|22224444 e\|jamesho@example.com a\|123, Clementi Rd, 1234665 t\|friend t\|colleague m\|A1234567X` |
+| **Edit**               | `edit INDEX [n\|NAME] [p\|PHONE_NUMBER] [e\|EMAIL] [a\|ADDRESS] [t\|TAG]… [m\|MATRICULATION_NUMBER] [s\|STUDIO] [r\|REFLECTION]​` <br><br>• e.g.`edit 2 n\|James Lee e\|jameslee@example.com m\|A1234567X`                                              |
+| **Delete**             | `delete INDEX` <br><br>• e.g. `delete 3`                                                                                                                                                                                                                |
+| **DeleteShown**        | `deleteShown`                                                                                                                                                                                                                                           |
```

Comment:
DeleteShown should be Delete Shown _(Persons(?))_

Anyways, should not be join since it should be reader-friendly.

Reply:
Agreed on this

Reply:
Thanks for pointing that out!